cinema/tests/test_actor_api.py

```diff
@@ -19,6 +19,9 @@ def test_actor_list_is_subclass(self):
         self.assertEqual(issubclass(ActorList, mixins.CreateModelMixin), True)
         self.assertEqual(issubclass(ActorList, generics.GenericAPIView), True)
 
+    def test_actor_list_is_not_subclass(self):
+        self.assertEqual(issubclass(ActorList, viewsets.GenericViewSet), False)
```

Comment:
```suggestion
        self.assertIs(issubclass(ActorList, viewsets.GenericViewSet), False)
```
If I'm not mistaken you can use this method to compare booleans